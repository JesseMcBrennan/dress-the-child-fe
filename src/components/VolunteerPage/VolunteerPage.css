.main-body-volunteer {
  color: #2d313d;
  font-size: 1rem;
  font-family: 'Source Sans Pro', sans-serif;
  padding: 0 2rem;
  color: #111111;
  overflow: scroll;
  overflow-wrap: normal;
  max-height: 40rem; }

.shopping-event-list {
  text-align: left;
  padding-left: 5%; }

.shopping-event-list-item {
  width: 38rem;
  font-weight: 600;
  list-style-type: circle;
  padding: 0.5rem 1.5rem;
  font-size: 1.5rem;
  text-align: center; }

.main-body-title {
  font-weight: 300;
  font-size: 3rem; }

.volunteer-card {
  overflow: scroll; }

.contact-msg {
  font-size: 1rem;
  margin: 0 3rem; }

@media only screen and (max-width: 1600px) {
  .main-body-volunteer {
    font-size: 1.6rem; } }

@media only screen and (max-width: 1100px) {
  .main-body-volunteer {
    font-size: 1.3rem; }
  .shopping-event-list-item {
    width: 70%; } }

@media only screen and (max-width: 800px) {
  .main-body-volunteer {
    font-size: 0.8rem; }
  .shopping-event-list-item {
    width: 70%; } }
