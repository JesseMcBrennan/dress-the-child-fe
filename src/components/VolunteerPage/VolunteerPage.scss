.main-body-volunteer {
  color: #2d313d;
  font-size: 2rem;
  font-family: 'Source Sans Pro', sans-serif;
  font-size: 1.8rem;
  padding: 0 2rem;
  color: #111111;
  overflow: scroll;
  overflow-wrap: normal;
  max-height: 40rem;
}

.shopping-event-list {
  text-align: left;
  padding-left: 5%;
}

.shopping-event-list-item {
  width: 38rem;
  font-weight: 600;
  list-style-type: circle;
  padding: 0.5rem 1.5rem;
}

.main-body-title {
  font-weight: 300;
  font-size: 3rem;
}

.volunteer-card {
  overflow: scroll;
}

.contact-msg {
  font-size: 1.5rem;
  margin: 0 3rem;
}
