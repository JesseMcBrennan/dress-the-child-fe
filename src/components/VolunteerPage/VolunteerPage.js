import React from 'react';
import './VolunteerPage.css';

const VolunteerPage = () => (
  <div className="display-card">
    <h1 className="main-title">Volunteer</h1>
    <hr className="horizontal-rule" />
    <p className="main-body-volunteer">
      We spend our annual budget each year during November and December by
      taking hundreds of children shopping.  We match each child with at least
      one volunteer to take him or her through the store, and assist in making
      good decisions.  Our shopping dates for 2018 are listed below:
    </p>
    <ul>
      <li className="shopping-event-list">
        Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
      </li>
      <li className="shopping-event-list">
        Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
      </li>
      <li className="shopping-event-list">
        Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
      </li>
      <li className="shopping-event-list">
        Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
      </li>
    </ul>
  </div>
);

export default VolunteerPage;
