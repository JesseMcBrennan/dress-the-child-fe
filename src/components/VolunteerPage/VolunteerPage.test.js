import React from 'react';
import { shallow } from 'enzyme';
import { VolunteerPage } from './VolunteerPage';

describe('VolunteerPage', () => {
  let mockLocation;
  let wrapper;
  beforeEach(() => {
    mockLocation = {
      pathname: '/volunteer'
    };
    wrapper = shallow(<VolunteerPage location={mockLocation} />);
  });
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
