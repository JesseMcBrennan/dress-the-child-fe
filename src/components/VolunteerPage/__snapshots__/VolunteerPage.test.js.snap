// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VolunteerPage should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <VolunteerPage
    location={
      Object {
        "pathname": "/volunteer",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="main-title"
        >
          Volunteer
        </h1>,
        <hr
          className="horizontal-rule"
        />,
        <div
          className="main-body-volunteer"
        >
          <p
            className="main-body-volunteer"
          >
            We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:
          </p>
          <ul
            className="shopping-event-list"
          >
            <li
              className="shopping-event-list-item"
            >
              Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
            </li>
            <li
              className="shopping-event-list-item"
            >
              Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
            </li>
            <li
              className="shopping-event-list-item"
            >
              Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
            </li>
            <li
              className="shopping-event-list-item"
            >
              Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
            </li>
          </ul>
          <p
            className="contact-msg"
          >
            Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information
          </p>
        </div>,
      ],
      "className": "volunteer-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Volunteer",
          "className": "main-title",
        },
        "ref": null,
        "rendered": "Volunteer",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "horizontal-rule",
        },
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="main-body-volunteer"
            >
              We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:
            </p>,
            <ul
              className="shopping-event-list"
            >
              <li
                className="shopping-event-list-item"
              >
                Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
              </li>
              <li
                className="shopping-event-list-item"
              >
                Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
              </li>
              <li
                className="shopping-event-list-item"
              >
                Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
              </li>
              <li
                className="shopping-event-list-item"
              >
                Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
              </li>
            </ul>,
            <p
              className="contact-msg"
            >
              Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information
            </p>,
          ],
          "className": "main-body-volunteer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:",
              "className": "main-body-volunteer",
            },
            "ref": null,
            "rendered": "We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
                </li>,
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
                </li>,
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
                </li>,
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
                </li>,
              ],
              "className": "shopping-event-list",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy",
                  "className": "shopping-event-list-item",
                },
                "ref": null,
                "rendered": "Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy",
                  "className": "shopping-event-list-item",
                },
                "ref": null,
                "rendered": "Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's",
                  "className": "shopping-event-list-item",
                },
                "ref": null,
                "rendered": "Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's",
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy",
                  "className": "shopping-event-list-item",
                },
                "ref": null,
                "rendered": "Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy",
                "type": "li",
              },
            ],
            "type": "ul",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information",
              "className": "contact-msg",
            },
            "ref": null,
            "rendered": "Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="main-title"
          >
            Volunteer
          </h1>,
          <hr
            className="horizontal-rule"
          />,
          <div
            className="main-body-volunteer"
          >
            <p
              className="main-body-volunteer"
            >
              We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:
            </p>
            <ul
              className="shopping-event-list"
            >
              <li
                className="shopping-event-list-item"
              >
                Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
              </li>
              <li
                className="shopping-event-list-item"
              >
                Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
              </li>
              <li
                className="shopping-event-list-item"
              >
                Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
              </li>
              <li
                className="shopping-event-list-item"
              >
                Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
              </li>
            </ul>
            <p
              className="contact-msg"
            >
              Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information
            </p>
          </div>,
        ],
        "className": "volunteer-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Volunteer",
            "className": "main-title",
          },
          "ref": null,
          "rendered": "Volunteer",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "horizontal-rule",
          },
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="main-body-volunteer"
              >
                We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:
              </p>,
              <ul
                className="shopping-event-list"
              >
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
                </li>
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
                </li>
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
                </li>
                <li
                  className="shopping-event-list-item"
                >
                  Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
                </li>
              </ul>,
              <p
                className="contact-msg"
              >
                Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information
              </p>,
            ],
            "className": "main-body-volunteer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:",
                "className": "main-body-volunteer",
              },
              "ref": null,
              "rendered": "We spend our annual budget each year during November and December by taking hundreds of children shopping.  We match each child with at least one volunteer to take him or her through the store, and assist in making good decisions.  Our shopping dates for 2018 are listed below:",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="shopping-event-list-item"
                  >
                    Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy
                  </li>,
                  <li
                    className="shopping-event-list-item"
                  >
                    Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy
                  </li>,
                  <li
                    className="shopping-event-list-item"
                  >
                    Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's
                  </li>,
                  <li
                    className="shopping-event-list-item"
                  >
                    Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy
                  </li>,
                ],
                "className": "shopping-event-list",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy",
                    "className": "shopping-event-list-item",
                  },
                  "ref": null,
                  "rendered": "Sunday, November 4th at 6pm (sign-in at 5:30pm) Old Navy",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy",
                    "className": "shopping-event-list-item",
                  },
                  "ref": null,
                  "rendered": "Sunday, November 18th at 6pm (sign-in at 5:30pm) Old Navy",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's",
                    "className": "shopping-event-list-item",
                  },
                  "ref": null,
                  "rendered": "Sunday, December 2nd at 5:00pm (sign-in at 4:30pm) Kohl's",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy",
                    "className": "shopping-event-list-item",
                  },
                  "ref": null,
                  "rendered": "Sunday, December 9th at 6pm (sign-in at 5:30) Old Navy",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information",
                "className": "contact-msg",
              },
              "ref": null,
              "rendered": "Please contact Doug (575.644.9469) or Matt (575.993.2331) for more information",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
